acquia_lift.module

```diff
@@ -35,14 +35,17 @@ function acquia_lift_page_attachments(array &$page) {
   $path_context->populate($page);
   $page_context->populate($page);
 
-  // Attach lift CSS to all pages.
-  $page['#attached']['library'][] = 'acquia_lift/acquia-lift';
+  // Attach lift CSS to all pages if the toolbar is present.
```

Comment:
Curious if we should rename the acquia_lift library to acquia_lift_toolbar, as this has only changes (also the CSS) to the toolbar stuff.